projects/apache.org/zookeeper/package.yml

```diff
@@ -0,0 +1,78 @@
+versions:
+  url: https://archive.apache.org/dist/zookeeper/
+  match: /zookeeper-\d+\.\d+\.\d+/
+  strip:
+    - /^zookeeper-/
+dependencies:
+  openjdk.org: '*'
+  openssl.org: '*'
+build:
+  dependencies:
+    gnu.org/autoconf: '*'
+    gnu.org/automake: '*'
+    freedesktop.org/cppunit: '*'
+    gnu.org/libtool: '*'
+    maven.apache.org: '*'
+    freedesktop.org/pkg-config: '*'
+    curl.se: '*'
+    linux:
+      gnu.org/gcc: '*'
+  # Malformed input or input contains unmappable characters: ~/pantry/builds/apache.org???zookeeper-3.9.1+linux
+  working-directory: ../zookeeper
+  script:
+    - curl -L "https://archive.apache.org/dist/zookeeper/zookeeper-{{version}}/apache-zookeeper-{{version}}.tar.gz" | tar -xz --strip-components=1
```

Comment:
this seems pretty messy (and could easily mess up changes in brewkit in the future); can we just fix whatever part of their build is complaining? had to do something similar for folly:

https://github.com/pkgxdev/pantry/blob/59c5d75d836d7cd8bf08a13e0f3073fec1c5ae4c/projects/facebook.com/folly/package.yml#L30-L41

Regardless, I'd be more inclined to do something like:
- include the distributable
- cd ..
- if test -e zookeeper; rm -rf zookeeper; done
- mv "apache.org*zookeeper" zookeeper